New machine to test Diggr

YvesdeVoghel opened the pull request.

YvesdeVoghel commented:
Hallo Ramses,
Zou je dit toelaten...?

R-VdP commented:
Uiteraard.

I added one commit to add an entry in `org-spec/ocb_tunnels`, because otherwise you will get an error upon installation. When the reverse tunnel service is enabled, we enforce that such an entry is present to protect us from accidentally removing an entry and rebuilding the corresponding host, thereby losing all access.
Given that we don't have the key yet before installation, we leave it empty (just an empty string `""`), and you will fill it in with the real key once the installation finished.

I already made the commit and added it to this PR, but like this you know why.

YvesdeVoghel commented:
Thanks!
________________________________
From: R-VdP <notifications@github.com>
Sent: Thursday, April 2, 2020 10:01:56 PM
To: MSF-OCB/NixOS <NixOS@noreply.github.com>
Cc: Yves de Voghel <Yves.de.Voghel@brussels.msf.org>; State change <state_change@noreply.github.com>
Subject: Re: [MSF-OCB/NixOS] New machine to test Diggr (#22)


Uiteraard.

I added one commit to add an entry in org-spec/ocb_tunnels, because otherwise you will get an error upon installation. When the reverse tunnel service is enabled, we enforce that such an entry is present to protect us from accidentally removing an entry and rebuilding the corresponding host, thereby losing all access.
Given that we don't have the key yet before installation, we leave it empty (just an empty string ""), and you will fill it in with the real key once the installation finished.

I already made the commit and added it to this PR, but like this you know why.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://github.com/MSF-OCB/NixOS/pull/22#issuecomment-608073698>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIPDEPFRHRHX7EW563XNPULRKTVLJANCNFSM4L2UHYHA>.
